lib/provider.coffee

```diff
@@ -94,13 +100,19 @@ module.exports =
       hasScope(previousScopesArray, 'entity.other.attribute-name.id') or
       hasScope(previousScopesArray, 'entity.other.attribute-name.parent-selector.css') or
       hasScope(previousScopesArray, 'entity.name.tag.reference.scss') or
-      hasScope(previousScopesArray, 'entity.name.tag.scss')
+      hasScope(previousScopesArray, 'entity.name.tag.scss') or
+      hasScope(previousScopesArray, 'entity.name.tag.reference.postcss') or
+      hasScope(previousScopesArray, 'entity.name.tag.postcss')
 
     isAtBeginScopePunctuation = hasScope(scopes, 'punctuation.section.property-list.begin.bracket.curly.css') or
       hasScope(scopes, 'punctuation.section.property-list.begin.bracket.curly.scss') or
+      hasScope(scopes, 'punctuation.section.property-list.begin.bracket.curly.postcss') or
+      hasScope(scopes, 'punctuation.section.property-list.begin.postcss') or
```

Comment:
Ok, i figured it out: yes, it's needed. Otherwise it fails two tests:

- "does not autocomplete property names when preceding a {"
- "does not autocomplete property names when immediately after a }"

I also attach a screenshot to prove it:

<img width="100%" alt="postcssscope" src="https://cloud.githubusercontent.com/assets/5115382/26531679/fb6e18cc-43ed-11e7-84a7-ceddd432e7ad.png">